go/vt/mysqlctl/schema.go

```diff
@@ -109,6 +112,15 @@ func (mysqld *Mysqld) GetSchema(ctx context.Context, dbName string, request *tab
 
 			fields, columns, schema, err := mysqld.collectSchema(ctx, dbName, td.Name, td.Type, request.TableSchemaOnly)
 			if err != nil {
+				// there's a possible race condition: it could happen that a table was dropped in between reading
```

Comment:
IF you have to do another push for some reason, we can capitalize `There's` here. It's annoyingly nitty, I know, so no need unless we have to do another commit/push for another reason.